plugin.video.jwtv-unofficial/addon.py

```diff
@@ -22,162 +22,158 @@
 if len('' + language) < 1: language = 'E'
 
 def build_url(query):
-	return base_url + '?' + urllib.urlencode(query)
+    return base_url + '?' + urllib.urlencode(query)
 
 def get_json(url):
-	data = urllib2.urlopen(url).read().decode('utf-8')
-	return json.loads(data)
+    data = urllib2.urlopen(url).read().decode('utf-8')
+    return json.loads(data)
 
 def b64_encode_object(obj):
-	js = json.dumps(obj)
-	b64 = base64.b64encode(js)
-	return b64
+    js = json.dumps(obj)
+    b64 = base64.b64encode(js)
+    return b64
 
 def b64_decode_object(str):
-	js = base64.b64decode(str)
-	obj = json.loads(js)
-	return obj
+    js = base64.b64decode(str)
+    obj = json.loads(js)
+    return obj
 
 def build_folders(subcat_ary):
-	isStreaming = mode[0] == 'Streaming'
-	for s in subcat_ary:
-		url = build_url({'mode': s.get('key')})
-		li = xbmcgui.ListItem(s.get('name'))
-		if 'rph' in s['images']:
-			li.setIconImage(s['images']['rph'].get('md'))
-			li.setThumbnailImage(s['images']['rph'].get('md'))
-		if 'pnr' in s['images']:
-			li.setProperty('fanart_image', s['images']['pnr'].get('md'))
-		xbmcplugin.addDirectoryItem(handle=addon_handle, url=url, listitem=li, isFolder=(isStreaming == False))
+    isStreaming = mode[0] == 'Streaming'
+    for s in subcat_ary:
+        url = build_url({'mode': s.get('key')})
```

Comment:
Hum probably better to initialize the dictionary and just add the keys if they exist. The way you're doing this right now basically replaces the dictionary